bigbluebutton-html5/imports/ui/components/video-preview/styles.scss

```diff
@@ -123,6 +123,10 @@
   }
 }
 
+.mirroredVideo {
+  transform: scale(-1, 1);
```

Comment:
It isn't a probelm, but I've a question, does it not affect the drag and drop of the webcam? the transform create a new z-index stack that could cause some issues.